src/main/kotlin/sliv/tool/common/Event.kt

```diff
@@ -0,0 +1,19 @@
+package sliv.tool.common
+
+class Event<T> {
+    private val listeners = mutableSetOf<(T) -> Unit>()
+
+    operator fun plusAssign(observer: (T) -> Unit) {
+        synchronized(this) {
```

Comment:
is it called from different threads?

Reply:
UPD: now LandmarkViews creates in the UI thread